libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -204,8 +204,10 @@ void AP_InertialSensor_Backend::_notify_new_gyro_raw_sample(uint8_t instance,
         _imu._last_delta_angle[instance] = delta_angle;
         _imu._last_raw_gyro[instance] = gyro;
 
-        // apply the low pass filter
-        _imu._gyro_filtered[instance] = _imu._gyro_filter[instance].apply(gyro);
+        // apply the low pass filters
+        _imu._gyro_filtered[instance] = _imu._gyro_filter2[instance].apply(gyro);
+        _imu._gyro_filtered[instance] = _imu._gyro_filter[instance].apply(_imu._gyro_filtered[instance]);
```

Comment:
have you done any tests for how this effects the phase and time delay? This second order filter may have some interesting effects when it hits the EKF.

Reply:
I think the EKF uses unfiltered delta angles, so I believe unaffected by this change.

Reply:
Also to be clear this replaces a 2-pass filter, with two 1-pass filters so I think time delay should be the same.